x-pack/legacy/plugins/siem/public/lib/keury/index.ts

```diff
@@ -84,12 +84,13 @@ export const convertToBuildEsQuery = ({
   filters: Filter[];
 }) => {
   try {
+    const core = useKibanaCore();
```

Comment:
Do you want this hook within the `try`? Insinuates it requires error handling?

Reply:
I will move it up, I was hesitating on that.